jira/utils/__init__.py

```diff
@@ -79,3 +82,79 @@ def json_loads(resp: Response | None) -> Any:
         if not resp.text:
             return {}
         raise
+
+
+def cloud(client_method: Callable) -> Callable:
```

Comment:
Part of me is nervous about such a simple name, but seeing it in use it does look fine. But I wonder if there are other names we can use

Reply:
Changed this to `cloud_api`. I think that's a little clearer hopefully.